cogs/release.py

```diff
@@ -0,0 +1,198 @@
+#un implemented
+
+from asyncio.windows_events import NULL
+import discord
+from discord import client
+from discord.ext.commands import Cog, command, has_any_role
+from discord.ext import commands
+from utils import load_config
+
+global_config = load_config()
+config = global_config
+
+# TWITTER KEYS (...would go here)
+consumer_key = None
+consumer_secret = None
+access_token = None
+access_token_secret = None
+
+#colors
+tf2_color = 0xB98A2E
+art_color = 0xde1f1f
+other_color = 0x868686
+
+class twitter(Cog):
+
+    @commands.command()
+    @has_any_role('Staff', 'Server Mods', 'Senior Staff', 'Fub')
+    async def release(self, ctx):
```

Comment:
do we want to put these in the bot channel or have this be an exclusively DM thing?

Reply:
you initiate it from a command in the bot channel, !release, it deletes the message to help prevent other users from being "why are you releasing this" then continues submission in DM. Still a WIP, probably shouldn't have pushed it just yet, my bad.